### DISCUSSION TITLE
Welcome to card-creatr-studio Discussions!

### DISCUSSION THREAD

**Discussion by sffc** [sffc/card-creatr-studio#56] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ASL4U** [comment]
Just downloaded the appimage and when I mark it executable - and the double click it - it opens my file directory asking me to locate a card creatr file. There is no interface. There is no program - just an open screen asking me to show it where to find a card creatr file.  I tried to figure this out about 6 months ago - but gave up - now I'm back. This should work. Tonight I went and downloaded an earlier version - hoping at least I'd get an interface - or something that would let me open a file - I got exactly the same window. My own file manager starting in the card creatr folder and looking for a file to open. ? What kind of file does this open and how do I great one for it to open so I can see how it works? Any ideas out there?

**Reply by doc-halo** [reply]
I had the same problem on linux. I finally figured out two workarounds: 

1. Download one the templates on the website: https://cardcreatr.sffc.xyz/templates/ and open it. From there you can either fool around with the template or choose to open a "New Empty File" from File.

2. Or once you start the AppImage, you can open a random file on your hard drive. It will give you an error. But from there you can still choose to open a "New Empty File" from File.

**Reply by mpowild** [reply]
When the file directory opens just hit `esc`, then `Cmd+N` (Mac) or `Ctrl+N` (Windows), or `File > New Empty File` to open a new file
